Solaris specific patch, not suitable for upstream.
The pytest.ini changes are required as the flake8 tool
is not available in Solaris.
--- tempora-1.12.old/pytest.ini	2018-06-26 06:01:55.000000000 +0000
+++ tempora-1.12/pytest.ini	2018-08-31 06:05:26.199003252 +0000
@@ -1,4 +1,4 @@
 [pytest]
 norecursedirs=dist build .tox .eggs
-addopts=--doctest-modules --flake8
+addopts=--doctest-modules
 doctest_optionflags=ALLOW_UNICODE ELLIPSIS
